packaging/ansible-runner-service-project/project/roles/ovirt-host-deploy-firewalld/tasks/main.yml

```diff
@@ -1,36 +1,36 @@
 ---
 - block:
     - name: Stop iptables service if running
-      service:
+      ansible.builtin.service: # noqa ignore-errors
```

Comment:
for this task- the task for running firewalld (2 tasks after this one) will fail anyway if iptables is running.
If iptables is stopped and the role can continue, and we don't care about iptables at this point- I don't see a reason to handle any errors related to iptables at this point.

host upgrade-
line #12: see my discussion with @almusil here-
https://github.com/oVirt/ovirt-engine/pull/413

line #46: this is not a mandatory part of the upgrade process. we added it to clear space before upgrading, and If cleaning cache fails, in most cases there'll be enough space to continue with upgrading the packages.

Reply:
> for this task- the task for running firewalld (2 tasks after this one) will fail anyway if iptables is running. If iptables is stopped and the role can continue, and we don't care about iptables at this point- I don't see a reason to handle any errors related to iptables at this point.

OK. Please put this information to the commit message. The current explanation there, "Use failed_when and specify error conditions instead of using ignore_errors", talks about something different and unrelated to the current version, which confused me. It has probably been forgotten to update when the patch changed?

> host upgrade- line #12: see my discussion with @almusil here- #413
> 
> line #46: this is not a mandatory part of the upgrade process. we added it to clear space before upgrading, and If cleaning cache fails, in most cases there'll be enough space to continue with upgrading the packages.

I'm losing the context here.